blog/forms.py

```diff
@@ -0,0 +1,9 @@
+# forms.py
+from django import forms

```

Comment:
Add a blank line between different import groups to follow PEP-8 here and in the other files as well